packages/tokens/focus-tokens/tokens.json

```diff
@@ -0,0 +1,57 @@
+{
+  "nl": {
+    "focus-backdrop": {
+      "background-color": {
+        "$extensions": {
+          "nl.nldesignsystem.css-property-syntax": "<color>",
+          "nl.nldesignsystem.figma-implementation": true
+        },
+        "$type": "color"
+      },
+      "color": {
+        "$extensions": {
+          "nl.nldesignsystem.css-property-syntax": "<color>",
+          "nl.nldesignsystem.figma-implementation": true
+        },
+        "$type": "color"
+      }
+    },
+    "focus-ring": {
+      "outline-width": {
+        "$extensions": {
+          "nl.nldesignsystem.css-property-syntax": "<length>",
+          "nl.nldesignsystem.figma-implementation": false
+        },
+        "$type": "borderWidth"
+      },
+      "outline-color": {
+        "$extensions": {
+          "nl.nldesignsystem.css-property-syntax": "<color>",
+          "nl.nldesignsystem.figma-implementation": false
+        },
+        "$type": "color"
+      },
+      "inverse-outline-color": {
```

Comment:
`inverse-outline-color` is geen bestaande CSS property, en volgt daarom niet onze Design Tokens conventie